src/Clock.jsx

```diff
@@ -195,7 +235,7 @@ export default function Clock({
       <svg
         fill="none"
         height="33"
-        style={{ transform: `translateX(calc(${size / 2}px - 50%)) translateY(calc(-${size / 8}px - 50%))` }}
+        style={{ transform: `translateX(calc(${size / 2}px - 50%)) translateY(calc(-${size / 8}px - 830%))` }}
```

Comment:
`- 830%` is pretty wild. What's going on here?

Reply:
Oh yeah.. I was just trying to position it where it used to be tbh, after I made the change it was at the bottom of the clock so I bumped this way up.
As far as I could tell the result was the same, but there'd be a way to do this with a smaller number than 830 (going from the other side, or absolutely centering first) if you think that's worth looking into?

Reply:
No need to block the release of this. Was mostly just curious